src/test/qtumtests/test_utils.cpp

```diff
@@ -65,3 +65,12 @@ std::pair<std::vector<ResultExecute>, ByteCodeExecResult> executeBC(std::vector<
     globalState->dbUtxo().commit();
     return std::make_pair(res, bceExecRes);
 }
+
+std::string bytesToHexString(unsigned char* muhBytes, int len)
```

Comment:
Pretty sure there is a function for this already in Qtum, but no idea where it would be located. Should be ok to leave for now though since it's only used in tests